airbyte-integrations/connectors/source-datadog/source_datadog/schemas/monitors.json

```diff
@@ -0,0 +1,365 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": true,
+  "properties": {
+    "created": {
+      "type": ["string", "null"]
+    },
+    "creator": {
+      "additionalProperties": true,
+      "properties": {
+        "email": {
+          "type": ["string", "null"]
+        },
+        "handle": {
+          "type": ["string", "null"]
+        },
+        "name": {
+          "type": ["string", "null"]
+        }
+      },
+      "type": ["object", "null"]
+    },
+    "deleted": {
+      "type": "null"
+    },
+    "id": {
+      "type": "integer"
+    },
+    "matching_downtimes": {
+      "items": {
+        "additionalProperties": true,
+        "properties": {
+          "end": {
+            "type": ["integer", "null"]
+          },
+          "id": {
+            "type": "integer"
```

Comment:
usually it is better to always define ["null", "integer"] or the data type to make the schema more robust.

Reply:
That's fair. I was going off of the documentation of what could be `null` or not. I'll change all of these accordingly.

Reply:
fixed